docs/content/getting-started/install.md

```diff
@@ -59,7 +59,7 @@ Run `hof help` in your terminal.
 ## Updating __hof__
```

Comment:
This section works for linux but not mac. I think we should separate the sections like install and add one for homebrew

Reply:
<img width="1227" alt="Captura de ecrã 2023-03-16, às 12 13 58" src="https://user-images.githubusercontent.com/29093946/225614052-ae8f96e7-f3ff-472e-89b9-c0111bbeaeaa.png">

what do you think of this?